manifests/crds/argoproj.io_applicationsets.yaml

```diff
@@ -1,13 +1,11 @@
+
```

Comment:
Renamed as this is where "make generate" is dropping the file, not sure why the mismatch.